src/pageLayout/containers/TreeNav.tsx

```diff
@@ -89,178 +88,176 @@ const TreeSidebar: FC<ReduxProps> = ({
   }
 
   return (
```

Comment:
tip: adding `?w=1` to the url in the file view will make this diff much less wacky. It removes whitespace changes, which is most of the lines of this diff.

https://github.com/influxdata/ui/pull/5328/files?w=1